fbfmaproom/README.md

```diff
@@ -1,66 +1,118 @@
 
 ```
-ssh -L 1234:localhost:1234 shortfin01.iri.columbia.edu
+ssh -L 1234:localhost:1234 shortfin.iri.columbia.edu
 ```
 
-### On the server, create an ssh key to use with GitHub
+## GitHub Account
+
+The repository is public, so unless you are making changes to the code, you don't need a github account.
+
+### On the server, create an ssh key to use with GitHub if you don't already have one.
 
 ```
-ssh-keygen -f ~/.ssh/id_rsa
+ssh-keygen -f ~/.ssh/id_rsa_github -t rsa
 ```
+
 Provide a strong password when prompted. Then
+
 ```
-cat ~/.ssh/id_rsa.pub
+cat ~/.ssh/id_rsa_github.pub
 ```
+
 and copy the output of that command for use in the next one. It should start with `ssh-rsa`.
 
 ### Configure GitHub to use that ssh key
-In a browser, visit [https://github.com/settings/ssh/new]. For the title put e.g. "IRI development server," and in the Key box paste the output from the previous comand. Then click the green "Add SSH key" button.
 
-### Clone this repository and enter the fbfmaproom directory
+In a browser, visit [https://github.com/settings/ssh/new](https://github.com/settings/ssh/new). For the title put e.g. "IRI development server," and in the Key box paste the output from the previous comand. Then click the green "Add SSH key" button.
+
+### Configure your IRI account to use that ssh key (Mac, Linux)
+
+```
+cat << EOF >> ~/.ssh/config
+
+Host github.com
+HostName github.com
+IdentityFile ~/.ssh/id_rsa_github
+PreferredAuthentications publickey
+IdentitiesOnly yes
+
+EOF
+chmod 600 ~/.ssh/config
+```
+
+## Clone this repository and enter the fbfmaproom directory
+
+If you have a github account and have set up your ssh key:
 
 ```
 git clone git@github.com:iridl/python-maprooms
 cd python-maprooms/fbfmaproom
 ```
 
+If you don't have a github account and will not be making any code changes:
+
+```
+git clone https://github.com/iridl/python-maprooms.git
+cd python-maprooms/fbfmaproom
+```
+
+## Python miniconda environment
+
 ### Enable conda
 
+This could be added to your ~/.bash_profile to be loaded every time you login.
+
 ```
-module load python/miniconda3.9.5
+module load python/miniconda3
+source /software/rhel8/x86_64/miniconda/3/etc/profile.d/conda.sh
 ```
 
-### Create a conda environment containing this project's dependencies
+### Optionally create a conda environment containing this project's dependencies
+
+On the IRI servers, this environment is pre-built, so skip this step unless you're using an environment where this doesn't exist.
+
+```
+cond env list
+```
+
+If fbmaproom2 doesn't exist:
 
 ```
 conda create -n fbfmaproom2 --file conda-linux-64.lock
 ```
-(substituting osx or win for linux as appropriate)
 
-You don't need to install conda-lock for this.
+(substituting conda-osx-64.lock or conda-win-64.lock appropriate)
+
+Do not use conda-lock for this.
 
 Note that the command is `conda create`, not `conda env create`. Both exist, and they're different :-(
 
 ### Create a local configuration file
```

Comment:
A pre-existing problem: this is part of the "Python miniconda environment" section, but has nothing to do with miniconda. Should be bumped out one level.